README.md

```diff
@@ -53,6 +53,10 @@ which is supported by the tldr-pages project maintainers:
 
     npm install -g tldr
 
+You can also use the fully-featued [official Python client](https://github.com/tldr-pages/tldr-python-client), which can be installed via `pip3`.
```

Comment:
What do you think?
```suggestion
You can also use the fully-featued, [official Python client](https://github.com/tldr-pages/tldr-python-client), which can be installed via `pip3`.
```

Reply:
```suggestion
You can also use the fully-featured [official Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
```

Reply:
maybe something like `You can also use, the ....`

Reply:
@nicokosi nice catch. I also agree on removing the comma before `which`. But what do you think about the comma in `fully-featured, official Python client`? 

@CleanMachine1 I don't think that's correct in any way.

Reply:
hmm, reading it again it doesn't make much sense.

Reply:
@nicokosi's suggestion makes the most grammatical sense to me.